tests/components/zwave_me/test_config_flow.py

```diff
@@ -0,0 +1,140 @@
+"""Test the zwave_me config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries
+from homeassistant.components import zeroconf
+from homeassistant.components.zwave_me.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import (
+    RESULT_TYPE_ABORT,
+    RESULT_TYPE_CREATE_ENTRY,
+    RESULT_TYPE_FORM,
+    FlowResult,
+)
+
+MOCK_ZEROCONF_DATA = zeroconf.ZeroconfServiceInfo(
+    host="192.168.1.14",
+    hostname="mock_hostname",
+    name="mock_name",
+    port=1234,
+    properties={
+        "deviceid": "aa:bb:cc:dd:ee:ff",
+        "manufacturer": "fake_manufacturer",
+        "model": "fake_model",
+        "serialNumber": "fake_serial",
+    },
+    type="mock_type",
+)
+
+
+async def mock_uuid(url, _):
+    """Mock getting uuid."""
+    if "192.168.1.14" in url:
+        return "12345678"
+    else:
+        return None
+
+
+@patch("homeassistant.components.zwave_me.get_uuid", mock_uuid)
+async def test_form(hass: HomeAssistant) -> None:
+    """Test we get the form."""
+    with patch(
+        "homeassistant.components.zwave_me.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry:
+        result = await hass.config_entries.flow.async_init(
+            DOMAIN, context={"source": config_entries.SOURCE_USER}
+        )
+        assert result["type"] == RESULT_TYPE_FORM
+        assert result["errors"] == {}
+        result2 = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            {
+                "url": "192.168.1.14",
+                "token": "test-token",
+            },
+        )
+        await hass.async_block_till_done()
+        assert result2["type"] == RESULT_TYPE_CREATE_ENTRY
+        assert result2["title"] == "ws://192.168.1.14"
+        assert result2["data"] == {
+            "url": "ws://192.168.1.14",
+            "token": "test-token",
+        }
+        assert len(mock_setup_entry.mock_calls) == 1
```

Comment:
We can assert the result outside the patch context manager.
```suggestion

    assert result2["type"] == RESULT_TYPE_CREATE_ENTRY
    assert result2["title"] == "ws://192.168.1.14"
    assert result2["data"] == {
        "url": "ws://192.168.1.14",
        "token": "test-token",
    }
    assert len(mock_setup_entry.mock_calls) == 1
```